### DISCUSSION TITLE
記事の更新停滞警告▲!は何を意味するものですか？

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#330] [要望・アイデア・不具合]
少し古い記事には、ページの最上段の目立つところに「▲ この記事は最終更新日から１年以上が経過しています。」のような警告が表示されます。しかし、これを表示する意図が良く分かりません。

　記事の中には、新鮮さだけがウリのようなものもあれば、恒久不変の情報を提供しているようなものもあります。記事の内容に関係なく、一律に「賞味期限切れの50%割引シール」のような表示をするのは何故でしょうか？

　表示をやめて欲しいという訳ではありませんが、一度、表示の意図を確認させていただきたく、ここに質問させていただきました。

　自分の記事にこの表示が付いているのに気づいたときには、記事の内容が陳腐化していないと分かっていても、一旦「編集する」をクリックし、上書き再投稿した方が良いのでしょうか？

**Comment by yoshinoyuya** [comment]
ツールや、プログラミング言語は、不変ではありません。
バージョンアップで、仕様が変わったり、今までは標準的だったことが、非推奨になったりします。

しかし、記事に関しては必ずアップデートされるわけではないため
問題が起きないための、注意喚起メッセージだと思います。

また、いいこともあるかと思います。
バグで動かないことを書いた記事でも、1年前のことが分かれば今は修正されているかもしれないという
気付きになるかと思います。

**Reply by tsubolabo** [reply]
確かに仰られるとおりです。どうも、私にとっては「▲！」マークが刺激的過ぎて、叱られているような感覚を持ってしまったようです。イエローカードを突き付けられているような感じです。このマークさえなければ、どうということはないのですが。

>ツールや、プログラミング言語は、不変ではありません。
バージョンアップで、仕様が変わったり、今までは標準的だったことが、非推奨になったりします。

そのとおりです。そこで、場違いなお願いですが・・・。
この点に関して、最近、Qiitaの「質問」に下記を投稿したのですが未だ回答がつきません。ご存じであれば、宜しくお願いします。

[あるコンパイラ言語の新旧の仕様の差について]
(https://qiita.com/tsubolabo/questions/8aa0710362400a23ba19)

**Reply by yoshinoyuya** [reply]
## 警告について
>確かに仰られるとおりです。どうも、私にとっては「▲！」マークが刺激的過ぎて、叱られているような感覚を持ってしまったようです。イエローカードを突き付けられているような感じです。このマークさえなければ、どうということはないのですが。

難しいところではありますね、現在は非推奨となり、場合によっては脆弱性となる可能性などもあるので
閲覧者には、「記事の記載時期」と「注意事項」がしっかり伝わり、記事の作成者には`マークが刺激的過ぎて、叱られているような感覚`に陥らない、代案があれば良いとは思うのですが

現在の仕様がシンプルかつ効果的ではあるので…

## Qiit質問
>[あるコンパイラ言語の新旧の仕様の差について]
(https://qiita.com/tsubolabo/questions/8aa0710362400a23ba19)

すみません。
分からないというのが回答となります。
当時勉強していた言語が分かれば、まだ特定しやすいとは思うのですが
1つの言語に絞るのが難しいのであれば、可能性がある言語を幾つかあげてみては、どうでしょうか

**Reply by tanaka-asami415** [reply]
@yoshinoyuya 
ありがとうございます🙏

@tsubolabo 
警告を表示している理由としては、yoshinoyuya さんご記載の通りです。
マークについては↓のようにWarningとして一般的なマークということもあり採用しております。ご理解いただけますと幸いです🙇
> **Warning**
https://github.com/community/community/discussions/16925

**Reply by tsubolabo** [reply]
この提言直後に、確かに、記事には鮮度が重要であることを身に染みて感じました。Qiitaから賞味期限の査定を受けるのも待てず、投稿の僅か５日後に、自ら期限切れの警告を出すことになってしまいました。
[【Qiita検索オプション】試行錯誤と応用技](https://qiita.com/tsubolabo/items/5ab55372701bffff8103)
